modules/core/help.py

```
@@ -0,0 +1,295 @@
+        if use_local:
+            pic = await download(webrender('element_screenshot', use_local=False),
+                                 status_code=200,
+                                 method='POST',
+                                 headers={'Content-Type': 'application/json'},
+                                 post_data=html_,
+                                 request_private_ip=True
+                                 )
+        else:
+            Logger.info('[Webrender] Generation Failed.')
+            return False
+    read = open(pic)
```

Ruff

Use a context manager for opening files

(SIM115)